formula/composer.rb

```diff
@@ -0,0 +1,16 @@
+class Composer < Formula
+  desc "Dependency Manager for PHP"
+  homepage "https://getcomposer.org"
+  url "https://github.com/composer/composer/releases/download/1.6.3/composer.phar"
+  sha256 "52cb7bbbaee720471e3b34c8ae6db53a38f0b759c06078a80080db739e4dcab6"
+
+  bottle :unneeded
+
+  def install
+    bin.install "composer.phar" => "composer"
+  end
+
+  test do
+    system "#{bin}/composer", "--version"
```

Comment:
We'll need a test that exercices some of the software's functionality…

Reply:
@SMillerDev any ideas on this one ?

Reply:
I'm thinking something like this should do it https://www.frobiovox.com/posts/2016/08/16/basic-hello-world-with-composer-and-php.html